CONTRIBUTING.md

```diff
@@ -22,10 +22,18 @@ and we use [rustfmt](https://github.com/rust-lang/rustfmt) to make our code clea
     ```shell
     cargo clippy -- -D warnings
     ```
+   Install cargo udeps
+   ```shell
+    cargo install cargo-udeps --locked
```

Comment:
What's the reason using `--locked` flag for installing binary crate to the rust's root dir ? :)

Reply:
Well, I used the installation description of these plugins from crates.io.